content/docs/csidriver/installation/operator/_index.md

```diff
@@ -266,25 +266,64 @@ The CSI Drivers installed by the Dell CSI Operator can be updated like any Kuber
 
 **NOTES:** 
 1. If you are trying to upgrade the CSI driver from an older version, make sure to modify the _configVersion_ field if required.
-2. The parameter "dnsPolicy: ClusterFirstWithHostNet" should be added as follows:
+2. Volume Health Monitoring feature is optional and by default this feature is disabled for drivers when installed via operator.
+   To enable this feature, we will have to modify the below block while upgrading the driver.To get the volume health state `value` 
```

Comment:
To get the volume health state `value - > To get the volume health state add external-health-monitor sidecar in the sidecar section and  `value` under controller...

Reply:
taken care